nomenclature/cli.py

```diff
@@ -93,3 +96,31 @@ def cli_valid_project(
     """
     assert_valid_yaml(path)
     assert_valid_structure(path, definitions, mappings, required_data, dimensions)
+
+
+@cli.command("run-region-processing")
+@click.argument("input_data_file", type=click.Path(exists=True, path_type=Path))
+@click.option(
+    "-d",
+    "--definitions",
+    type=click.Path(exists=True, path_type=Path),
+    default="definitions",
+)
+@click.option(
+    "-m", "--mappings", type=click.Path(exists=True, path_type=Path), default="mappings"
+)
+@click.option("--export-processing-result", is_flag=True, default=True)
+@click.option("--export-differences", is_flag=True, default=False)
```

Comment:
Maybe make that a string or `None`, where the string is the name of the output file (default `None`, no file is exported.

Reply:
Done